src/Websites/Websites.Test/ScenarioTests/WebAppTests.ps1

```diff
@@ -1321,6 +1321,68 @@ function Test-WebAppPublishingProfile
 	}
 }
 
+function Test-PublishAzureWebAppFromZip
+{
+	# Setup
+	$rgname = Get-ResourceGroupName
+	$appName = Get-WebsiteName
+	$location = Get-WebLocation
+	$planName = Get-WebHostPlanName
+	$tier = "Shared"
+
+	try
+	{
+		#Setup
+		New-AzureRmResourceGroup -Name $rgname -Location $location
+		$serverFarm = New-AzureRmAppServicePlan -ResourceGroupName $rgname -Name  $planName -Location  $location -Tier $tier
+		
+		# Create new web app
+		$webapp = New-AzureRmWebApp -ResourceGroupName $rgname -Name $appName -Location $location -AppServicePlan $planName 
+		
+		$publishedApp = Publish-AzWebApp -ResourceGroupName $rgname -Name $appName -ArchivePath ".\ScenarioTests\Resources\nodejs-sample.zip"
+
+		Assert-NotNull $publishedApp
+	}
+	finally
+	{
+		# Cleanup
+		Remove-AzureRmWebApp -ResourceGroupName $rgname -Name $appName -Force
+		Remove-AzureRmAppServicePlan -ResourceGroupName $rgname -Name  $planName -Force
+		Remove-AzureRmResourceGroup -Name $rgname -Force
+	}
+}
+
+function Test-PublishAzureWebAppFromWar
+{
+	# Setup
+	$rgname = Get-ResourceGroupName
+	$appName = Get-WebsiteName
+	$location = Get-WebLocation
+	$planName = Get-WebHostPlanName
+	$tier = "Shared"
+
+	try
+	{
+		#Setup
+		New-AzureRmResourceGroup -Name $rgname -Location $location
+		$serverFarm = New-AzureRmAppServicePlan -ResourceGroupName $rgname -Name  $planName -Location  $location -Tier $tier
+		
+		# Create new web app
+		$webapp = New-AzureRmWebApp -ResourceGroupName $rgname -Name $appName -Location $location -AppServicePlan $planName 
+		
+		$publishedApp = Publish-AzWebApp -ResourceGroupName $rgname -Name $appName -ArchivePath ".\ScenarioTests\Resources\HelloJava.war"
```

Comment:
Please use "Join-Path" to create paths - this will ensure the cmdlet works on Linux and Windows (example here: https://github.com/Azure/azure-powershell/commit/449548cf55dd6cc22adb2f84091f5ada52bf763a#diff-6867939ded98d2200250bdd819564971R89)